src/shared/generator.c

```diff
@@ -191,3 +191,18 @@ int generator_write_timeouts(
                                     "[Unit]\nJobTimeoutSec=%s",
                                     program_invocation_short_name, timeout);
 }
+
+int generator_write_initrd_root_device_deps(const char *dir, const char *what)
+{
+        _cleanup_free_ char *unit = NULL;
```

Comment:
please follow coding style. opening bracket should be on same line as function name...